doc/guides/test/sanitycheck.rst

```diff
@@ -19,6 +19,26 @@ environment, but it does sufficient testing by building samples and
 tests for different boards and different configurations to help keep the
 full code tree buildable.
 
+sanitycheck's console output doesn't show any difference between a test
+that ran successfully versus a test that was just compiled
+successfully. There are a few reasons why sanitycheck will only build
+a test and not run it:
+
+- The test is marked as ``build_only: true`` in its ``.yaml``
+  configuration files. This is the case for a number of application code
+  examples in the ``samples/`` directory; they can be run but not from
+  sanitycheck.
+- The test configuration has defined a ``harness`` but you don't have
+  it or haven't set it up.
+- The script that invokes sanitycheck somewhere in the continuous
+  integration or other test automation of your project passes
+  ``--build-only``.
+
+There are at least three, non-mutually exclusive options that show which
+tests were merely compiled and not run: and ``-v -v`` (twice),
```

Comment:
```suggestion
tests were merely compiled and not run: ``-v -v`` (twice),
```

Reply:
Oops, that was caused by a late re-ordering. Thx for catching this.